### PR TITLE
Add/no acr

### PR DESCRIPTION
**Pull request by yosef32**
Gives the option to run the script with the choice to use ACR or not.
Login to ACR by username and password entered in environment variables (deploy.env)

### CODE REVIEW

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
How can you pull the images without registering to the repositoy?

File: deployment.sh [hunk 1]
```diff
@@ -171,11 +173,10 @@ fi
...
 ## -------
 # 5. Login to azure and set the azure conatiner registry
-msg "Logging Into Azure"
-az login
-msg "Logging Into Acr"
-az acr login --n $AZURE_CONTAINER_REGISTRY_NAME
-
+if ($ACR); then
+    msg "Logging Into Azure"
+    docker login  drivehub.azurecr.io -u $ACR_USER -p $ACR_PASS
```
Comment (nettashviro) [inline deployment.sh:178]: i'll prefer using "az login", the credentials stays for 3 days and you only need to call az acr login. why are you using this command?

Comment (nettashviro) [inline deployment.sh:178]: and also use env variables (AZURE_LOGIN_SERVER)

**Review by nettashviro — CHANGES_REQUESTED** [review]

File: deployment.sh (hunk 1 above)
Comment (yosef32) [reply deployment.sh:178]: It does not need a person for this operation, for az login volume that needs to be confirmed by user clicks

Comment (yosef32) [reply deployment.sh:178]: this user and pass is specific to the acr

File: README.md [hunk 2]
```diff
@@ -47,5 +47,7 @@ KBS_DNS="kbs-yaron"             # The name of the k8s dns
 HELM_DEPLOY_NAME="yaron-deploy" # Helm deployment name
 JSON_FILE="services.dev.json"   # name of json services file
 HELM_DEPENDENCIES=true          # reinstall helm dependencies
+ACR_USER="DriveHub"             # acr username from drivehub Access keys in azure
```
[the remark this answers is not in the document]
Comment (yosef32) [reply README.md]: done

**Review by Shahar-Y — APPROVED** [review]
Good job, make sure it is pushed to master accordingly

File: services.dev.json [hunk 3]
```diff
@@ -6,17 +6,13 @@
...
-        "tag": "develop"
-    },
     {
         "name": "download-service",
         "tag": "develop"
     },
     {
         "name": "drive-ui",
-        "tag": "develop"
+        "tag": "upload_folder"
```
Comment (nettashviro) [inline services.dev.json:15]: in develop json it should always stays in develop tag. if you want to change the json to your local environment you should use services.local.json

File: z-helm/drive-ui/values.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  tag: "v3.0.4"
+  tag: "upload_folder"
```
Comment (nettashviro) [inline z-helm/drive-ui/values.yaml:8]: same as my previous comment

**Review by nettashviro — CHANGES_REQUESTED** [review]

File: deployment.sh (hunk 1 above)
Comment (nettashviro) [reply deployment.sh]: the server url is the same for every user

**Pull request merged**